Readme

```diff
@@ -24,8 +24,8 @@ Installation
 Pippi requires almost no installation.
 
 1. Make sure you have the following already installed:
-   * Either Python v2.7 or later or Python v3
-   * NumPy and SciPy libraries for python (NumPy v0.9.0 or
+   * Either Python v2.7 or later or Python v3.8 or later
```

Comment:
Does the use of packaging for versioning break compatibility with Python 2.7?  If so, we should just remove 2.7 here, there's no real need to support it any longer.